### DISCUSSION TITLE
Batch  / bulk checks creation

### DISCUSSION THREAD

**Discussion by pmatuszy** [healthchecks/healthchecks#794] [General]
Hi
I have to create ~100 checks. They are 4 checks / server so it would be nice to be able to duplicate events but NOT clicking on this 
![image](https://user-images.githubusercontent.com/58336955/219875312-570ef2e4-dce2-401a-881b-9555399a2e4f.png)
but maybe by importing XML/JSON/CSV. 

It would help a lot. Maybe there is a way to do it in more clever way but I am not aware of it...
Any guidance / suggestions are welcome
Paul

**Comment by pmatuszy** [comment]
I wonder if it is even possible to get INSERTs into the database to create events... It would be even better...

**Comment by cuu508** [comment]
You can use [Management API](https://healthchecks.io/docs/api/#create-check) to mass-create checks.

[Here's](https://healthchecks.io/docs/cloning_checks/) a python snippet which reads checks from one project and recreates them in another project. You could edit the script to read check definitions from a JSON file or another source.

**Comment by pmatuszy** [comment]
Thank you - this worked like charm! Thank you for this clarification
